analyses/fastqc-analysis/run-fastqc-analysis.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+# Set up modules
+module load fastqc/0.11.5
+module load multiqc/1.0dev
+
+# set up running directory
+cd "$(dirname "${BASH_SOURCE[0]}")" 
+
+# Create results dir
+mkdir results
+mkdir results/01-fastqc-reports
+
+################################################################################################################
+# Run FastQC per library
+
+fastqc -o results/01-fastqc-reports /research/dept/hart/PI_data_distribution/zakhagrp/GSF/zakhagrp_321555_10XsNucRNAseq-1/*/*R2*.fastq.gz
```

Comment:
I'm not sure what stage is correct to remove HPC file paths, but I did want to note the run-fastqc-analysis.sh script does have a specific file path for the fastqc command, if that needs to be resolved now.